lib/yt/collections/reports.rb

```diff
@@ -136,7 +136,7 @@ def within(days_range, country, state, dimension, videos, try_again = true)
           elsif dimension.in? [:traffic_source, :country, :state, :playback_location, :device_type, :operating_system, :subscribed_status]
             hash = hash.transform_values{|h| h.sort_by{|range, v| -v}.to_h}
           end
-          (@metrics.one? || @metrics.keys == [:earnings, :estimated_minutes_watched]) ? hash[@metrics.keys.first] : hash
+          (@metrics.one? || @metrics.keys == [:estimated_revenue, :estimated_minutes_watched]) ? hash[@metrics.keys.first] : hash
```

Comment:
@kangkyu apart from testing calls like `video.earnings`, did you also try `video.reports(only: [:earnings])`? would that type of call also work?